package.json

```diff
@@ -239,6 +241,16 @@
                 "command": "azureFunctions.installOrUpdateFuncCoreTools",
                 "title": "%azFunc.installOrUpdateFuncCoreTools%",
                 "category": "Azure Functions"
+            },
+            {
+                "command": "azureFunctions.createSlot",
+                "title": "%azFunc.createSlot%",
+                "category": "Azure Functions"
```

Comment:
Do you care that these will show in command palette even with enableSlots=false?

Reply:
Yeah I do. Fixed thanks